src/containers/Search.js

```diff
@@ -1,9 +1,71 @@
 import React, { Component } from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, ScrollView, Alert, ListView, Platform  } from 'react-native';
 import { Header } from 'react-native-elements';
+import registerForPushNotificationsAsync from '../../api/registerForPushNotificationsAsync';
+import { Notifications } from 'expo';
+import * as firebase from 'firebase';
+import { List, ListItem, Button } from 'react-native-elements';
 
 export default class Search extends Component {
+  static navigationOptions = {
+    title: 'Links',
+  };
+  constructor(props) {
+    super(props);
 
+    this.state = {
```

Comment:
Before merging this, let's hook it up to Redux so we can simplify and centralize the state management.